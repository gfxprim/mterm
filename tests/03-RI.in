5 5
a\eMa\eMa\eMa\eMa\eM
\eM\eM\eM\eM\eM
\r
\eM\eM\eM\eM\eM
aaaaa\n
