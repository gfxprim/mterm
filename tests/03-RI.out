 -----
|aaaaa|
|     |
|     |
|     |
|     |
 -----
size 5x5 cursor 2x0
